2.0/m_pretenduser.cpp

```diff
@@ -62,10 +62,18 @@ class CommandPretendUser : public Command
 			return CMD_FAILURE;
 		}
 
-		if (IS_OPER(u))
+		if (IS_OPER(u) && IS_OPER(user))
 		{
-			user->WriteServ("NOTICE %s :*** Cannot target an IRC operator", user->nick.c_str());
-			return CMD_FAILURE;
+			std::string level = u->oper->getConfig("level");
```

Comment:
Rather than storing the level in a temporary variable and then calling atoi on its `.c_str()` you should just call ConvToInt here and store directly into dest_level and source_level.

Reply:
The strange thing is that m_operlevels.so store in a temp variable and uses longs. They are longs so I took the assumption it would be wrong to use integers here.

Reply:
Yeah, operlevels does that in 2.0 but it has been changed for 3.0.